### PR TITLE
fix badge display error

### PR DESCRIPTION
**Pull request by matthias-luger**
fix sold badge

### CODE REVIEW

File: src/components/Search/Search.tsx [hunk 1]
```diff
@@ -97,7 +97,7 @@ function Search(props: Props) {
         if (!props.selected || props.currentElement) {
             return props.currentElement || <div/>;
         }
-        return <p className="current"><Badge variant="primary">Current:</Badge> <img crossOrigin="anonymous" src={props.selected.iconUrl} width="32" height="32" alt="" style={{ marginRight: "10px" }} loading="lazy" />{props.selected.name || convertTagToName((props.selected as Item).tag)}</p>
+        return <p className="current"><img crossOrigin="anonymous" src={props.selected.iconUrl} width="32" height="32" alt="" style={{ marginRight: "10px" }} loading="lazy" />{props.selected.name || convertTagToName((props.selected as Item).tag)}</p>
```
Comment (Ekwav) [inline src/components/Search/Search.tsx:100]: not part of the breaking issue

File: src/api/WebsocketHelper.tsx [hunk 2]
```diff
@@ -48,9 +48,13 @@ function initWebsocket(): void {
     }
 
     let _handleSubscriptionOnMessage = function (response: ApiResponse, subscription: ApiSubscription) {
-        let parsedResponse = JSON.parse(response.data);
+        let parsedResponse = response.data;
+        try {
+            parsedResponse = JSON.parse(response.data);
+        }catch (e) {}
```
Comment (Flou21) [inline src/api/WebsocketHelper.tsx:54]: empty catch block

File: src/components/Flipper/Flipper.tsx [hunk 3]
```diff
@@ -178,17 +179,15 @@ function Flipper() {
...
-                                </p>
+                                <span className="card-label">Estimated Profit: </span><br />
+                                <span style={{ color: "green" }}>
+                                    +{numberWithThousandsSeperators(flipAuction.median - flipAuction.cost)} Coins
+                                </span>
+                                <div style={{ float: "right" }}>
+                                    <Tooltip tooltipTitle={<span>Auctions used for calculating the median price</span>} size="xl" type="click" content={<HelpIcon />}
+                                        tooltipContent={<FlipBased flip={flipAuction} />}
+                                    />
+                                </div>
```
Comment (Ekwav) [inline src/components/Flipper/Flipper.tsx:190]: what has this todo with the sold badge?

File: src/components/Flipper/Flipper.tsx [hunk 4]
```diff
@@ -129,14 +132,12 @@ function Flipper() {
...
-                    </div>
-                    {flipAuction.bin ? <Badge style={{ marginLeft: "5px" }} variant="success">BIN</Badge> : ""}
-                    {flipAuction.sold ? <Badge style={{ marginLeft: "5px" }} variant="danger">SOLD</Badge> : ""}
-                </p>
+                <div className="ellipse" style={{ width: flipAuction.bin && flipAuction.sold ? "60%" : "80%", float: "left" }}>
+                    <img crossOrigin="anonymous" src={flipAuction.item.iconUrl} height="24" width="24" alt="" style={{ marginRight: "5px" }} loading="lazy" />
+                    <span style={{ color: "lightgrey" }}>{flipAuction.item.name}</span>
+                </div>
+                {flipAuction.bin ? <Badge style={{ marginLeft: "5px" }} variant="success">BIN</Badge> : ""}
+                {flipAuction.sold ? <Badge style={{ marginLeft: "5px" }} variant="danger">SOLD</Badge> : ""}
```
Comment (Ekwav) [inline src/components/Flipper/Flipper.tsx:140]: what has this todo with the sold badge?

File: src/components/AuctionDetails/AuctionDetails.tsx [hunk 5]
```diff
@@ -257,7 +257,7 @@ function AuctionDetails(props: Props) {
 
     return (
         <div className="auction-details">
-            <Search currentElement={<p className="current"><Badge variant="primary">Current:</Badge> Auction Details</p>} />
+            <Search />
```
Comment (Ekwav) [inline src/components/AuctionDetails/AuctionDetails.tsx:260]: not part of the breaking issue

**Review by Ekwav — CHANGES_REQUESTED** [review]

**Comment by Ekwav** [comment]
As this is a [breaking] issue and does apparently work it is getting deployed

**Pull request merged**